logstash-core/spec/logstash/pipeline_dlq_commit_spec.rb

```diff
@@ -68,6 +68,7 @@ def close() end
 
   after(:each) do
     FileUtils.remove_entry pipeline_settings["path.dead_letter_queue"]
+    pipeline_settings_obj.reset
```

Comment:
I was simply following what was done in pipeline_spec.
OTOH does clone bring any previous settings with it?